src/hooks/useRole.js

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
+import userService from '../user/userService';
+
+const useRole = () => {
+  const [role, setRole] = useState(null);
```

Comment:
i have been using undefined as my default state, idk whats better

Reply:
eh i think its the same with the way we're using it